### ISSUE TITLE
Can I download an apk somewhere?

### ISSUE THREAD

**Issue by chmac** [NoahY/q2android#2]
I see a few forks of this repo, do I need to fork and hardcore the URL of my install, so then I'd be able to publish the app, or can I download it somewhere and just enter my URL into the app itself?

Thanks for sharing the app, it looks like it'll be an awesome addition to q2a. :+1:

**Comment by arjunsuresh** [comment]
yes. I just did that. No need to fork, just download a clone and hardcode the URL.
